routes.yaml

```diff
@@ -0,0 +1,7 @@
+app_name: bankrupt-officer-search-web
+group: web
+weight: 991
+# When we move static files to the CDN routes 3, and 4 will be removed
```

Comment:
This comment can be removed. There are no routes 3 and 4.

Reply:
Thanks @jsimons1989 done!